tests/test_iop_license/test_iop_license.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from hepcrawl.extractors import iop_parser
+import os
+
+from scrapy.selector import Selector
+
+correct_licenses = {
+    "abf13a.xml": [
+        {"url": "http://creativecommons.org/licenses/by/3.0/", "license": "CC-BY-3.0"}
+    ],
+    "ac3fab.xml": [
+        {"url": "http://creativecommons.org/licenses/by/3.0/", "license": "CC-BY-3.0"}
+    ],
+    "abb4d6.xml": [
+        {"url": "http://creativecommons.org/licenses/by/3.0/", "license": "CC-BY-3.0"}
```

Comment:
what about other type of licenses?

Reply:
we should not have any other license.  As I talked with Anne, she said if the license is different than cc-by, it means that the article is not scoap3. Maybe I should write this down somewhere? For future developers of scoap3